scripts/oecd/regional_demography/stats.txt

```diff
@@ -184,3 +197,13 @@ wikidataId  wikidataId/Q10686       2
             wikidataId/Q702083      2
             wikidataId/Q856010      2
 dtype: int64
+
+4. Number of resolved DCIDs by namespace (ignoring N to 1 issues):
+wikidataId    832
+nuts          748
+nan           371
+geoNamesId    274
+iso           266
+country        76
+geoId          53
+Name: namespace, dtype: int64
```

Comment:
move this up? 1. and this are basically what we look at. And downstream logic should need to resolve the N to 1 issues. (Do OECD scripts do this?)

Reply:
Moved. Not sure if you meant for me to add `1. and this are basically what we look at.` I interpreted as "move this up? this is basically what we look at. " If what I changed is OK, feel free to merge this PR, then I will sync and update #290

Reply:
This is ok. I meant #1 and <this-one> are the two stats to diff for validation. This is good, thanks.